org/all-prs.js

```diff
@@ -2,20 +2,22 @@
 
 const {
```

Comment:
is it important to have a separate line for each property?

Reply:
Having a single line for destructuring here makes this exceed line length linting rule, do you recommend another style?

Reply:
it does not need to be a single line, we just use line breaks before the limit.

Reply:
Done manually, would love for a github plugin or `eslint --fix` that did that though!